homeassistant/components/flo/device.py

```diff
@@ -40,7 +40,7 @@ def __init__(
     async def _async_update_data(self):
         """Update data via library."""
         try:
-            async with timeout(10):
+            async with timeout(20):
                 await asyncio.gather(
```

Comment:
Since you are having timeouts with this, does it make sense to make it do one at a time instead of a gather?

Reply:
The timeouts are very infrequent from what I have seen but I’m not opposed to splitting them up if you think it makes sense?

Reply:
I've seen cases where 

- the webserver on embedded devices can't handle the additional concurrency and drops requests which lead to timeouts.
- a cloud service blocks too many requests at the same time because of WAF rules

Not sure if thats what is going on here without an `strace`

Reply:
I don’t think it’s either. I think it’s just poor performance occasionally from the cloud service because it works 99% of the time. If it was a WAF I’d expect many more failures. 

I don’t mind splitting them. I assume each should have its own timeout when split?

Reply:
I think its fine to have a single timeout unless you expect one of them is more likely to timeout than the other or you need to know which call timed out

Reply:
gather removed